www/docs/customization/release.md

```diff
@@ -289,6 +290,10 @@ release:
   # You can disable this pipe in order to not upload any artifacts.
   disable: true
 
+  # If set to true, will not auto-publish the release.
+  # Available only for GitHub and Gitea.
+  draft: true
+
```

Comment:
this prop is already documented above

Reply:
The duplicate piece above was under GitHub section, but perhaps repeating it for Gitea explicitly is not worth it. Applied your suggestion!